oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentStoreStatsCollector.java

```diff
@@ -85,6 +85,19 @@ void doneQuery(long timeTakenNanos, Collection<? extends Document> collection, S
     void doneFindAndModify(long timeTakenNanos, Collection<? extends Document> collection, String key,
                            boolean newEntry, boolean success, int retryCount);
 
+    /**
+     * Called when a update operation was completed which could have affected multiple
+     * documents.
+     * @param timeTakenNanos time taken
+     * @param collection the collection
+     * @param ids List of document ids which got updated
+     * @param newEntry true if the document was newly created due to given operation
```

Comment:
Parameter `newEntry` looks strange to me. This method is about multiple documents. The only call to this method passes `false` and the new 'bulk' method DocumentStore.findAndUpdate() does not allow upsert or insert, right? I think the parameter should be removed.